cmd/kube-proxy/app/server.go

```diff
@@ -549,16 +578,6 @@ func createClients(config componentbaseconfig.ClientConnectionConfiguration, mas
 func (s *ProxyServer) Run() error {
```

Comment:
I still find it a little awkward that Run() cannot return a nil error. Should we document this somewhere or can we at least add a TODO comment here to clean this up in a follow-up PR?